packages/components/TranscriptCard/index.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import Alert from 'react-bootstrap/Alert';
+import Spinner from 'react-bootstrap/Spinner';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faTrash,
+  faCheck,
+  faExclamationTriangle,
+  faPen
+} from '@fortawesome/free-solid-svg-icons';
+
+const TranscriptCard = ({ transcriptItem, handleEdit, handleDelete }) => {
+
+  const handleDeleteClick = () => {
+    const confirmDeleteText = "Click OK if you wish to delete or cancel if you don't";
+    const cancelDeleteText = 'All is good, it was not deleted';
+
+    const confirmationPrompt = confirm(confirmDeleteText);
+
+    if (confirmationPrompt) {
+      return handleDelete ? handleDelete(transcriptItem.id) : alert(cancelDeleteText);
+    }
+  };
+
+  const handleEditClick = () => {
+    handleEdit(transcriptItem.id);
+  };
+
+  const setStatus = () => {
+    if (transcriptItem.status === 'error') {
+      return 'danger';
+    };
+    if (transcriptItem.status === 'in-progress') {
+      return 'info';
+    };
+    if (transcriptItem.status === 'done') {
+      return 'success';
+    };
+  };
+
+  const status = setStatus();
+
+  const setDescription = () => {
+    switch (status) {
+    case 'info':
+      return <Badge variant="info">In progress</Badge>;
+    case 'success':
+      return <Badge variant="success">Success</Badge>;
+    case 'danger':
+      return (
+        <>
+          <Alert variant="danger">
+            <FontAwesomeIcon icon={ faExclamationTriangle } />{' '}
+            {transcriptItem.errorMessage}
+          </Alert>
+          <Badge variant="danger">Error</Badge>
+        </>
+      );
+    }
+  };
+
+  const setStatusIcon = () => {
+    switch (status) {
+    case 'info':
+      return (
+        <Button variant="info" size="sm" disabled>
+          <Spinner
+            as="span"
+            animation="border"
+            size="sm"
+            role="status"
+            aria-hidden="true"
+          />
+        </Button>
+      );
+    case 'danger':
+      return (
+        <Button variant="danger" size="sm" disabled>
+          <FontAwesomeIcon icon={ faExclamationTriangle } />
+        </Button>
+      );
+    case 'success':
+      return (
+        <Button variant="success" size="sm" disabled>
+          <FontAwesomeIcon icon={ faCheck } />
+        </Button>
+      );
+    }
+  };
+
+  const description = setDescription();
+  const statusIcon = setStatusIcon();
+  const borderStatus = status && status === 'danger' ? 'danger' : null;
+  const title = status && status === 'success' ? <a href={ `${ transcriptItem.url }` }> {transcriptItem.title}</a> : transcriptItem.title;
+
+  return (
+    <Card
+      border={ borderStatus }
```

Comment:
```suggestion
      border={ status === 'danger' ? status : null }
```

Reply:
maybe that won't work as is, but you get the idea :)

Reply:
I suggest this to keep code consistent with everything else - eg if we are doing tertiary expressions, maybe we want to continue doing that all the way, as long as the tertiary expressions aren't complicated.

This is just a suggestion, doesn't actually affect the code.

Reply:
Cool beans - have implemented :)